subworkflows/local/reference_indexing.nf

```diff
@@ -17,20 +18,77 @@ workflow REFERENCE_INDEXING {
 
     // Warn user if they've given a reference sheet that already includes fai/dict/mapper index etc.
     if ( ( fasta.extension == 'csv' || fasta.extension == 'tsv' ) && (fasta_fai || fasta_dict || fasta_mapperindexdir)) log.warn("A TSV or CSV has been supplied to `--fasta` as well as e.g. `--fasta_fai`. --fasta CSV/TSV takes priority and --fasta_* parameters will be ignored.")
+    if ( ( fasta.extension == 'csv' || fasta.extension == 'tsv' ) && (params.mitochondrion_header || params.contamination_estimation_angsd_hapmap || params.damage_manipulation_pmdtools_reference_mask || params.snpcapture_bed || params.mapstats_bedtools_featurefile )) log.warn("A TSV or CSV has been supplied to `--fasta` as well as individual reference-specific input files, e.g. `--contamination_estimation_angsd_hapmap`. Input files specified in the --fasta CSV/TSV take priority and other input parameters will be ignored.")
```

Comment:
Someone pointed out ot me recently a possibly more concise version of the very long or

```
[params.mitochondrion_header, params.contamination_estimation_angsd_hapmap, params.damage_manipulation_pmdtools_reference_mask, params.snpcapture_bed, params.mapstats_bedtools_featurefile].anyOf()
```

(I think it's anyOf())

Minor but does the same thing so not necessary

Reply:
Finding it hard to find any documentation online, but will find out for the future :)

Reply:
It's groovy rather than nextflow, and I ment `any()`

For reference: try these different lines in `nextflow console`

```
[ true, false, false ].any()
[ false, false, false ].any()
[ false, false, false ].any()

[ 'tsv', 'csv'].contains('tsv')
[ 'tsv', 'csv'].contains('xsv')
```